help/destinations/catalog/advertising/acxiom-prospect-suppression.md

```diff
@@ -0,0 +1,180 @@
+* **File Export Options** - Defines if there should be a full export of the file, or should only include records that have been updated since the last run.  In either case the first run would run as a full export.
+* **Frequency** - Indicates how often the flow should be run hourly, daily.  Or simply to only run once. 
+  * When the File Export Options is set as 'Export full files' this will offer the 'Once' and 'Daily', this also offers two different scheduling types
+    * **After segment evaluation** - Activation runs immediately after the daily segmentation job completes. This ensures that the most up-to-date profiles are exported.
+    * **Scheduled** - Activation runs at a fixed time of the day. 
+  * When the File Export Options is set as 'Export incremental files' this will offer the 'Daily' and 'Hourly' with 3, 6, 8, and 12 hour increments.
+* **Start Time** - The timestamp for the projected run, presented in UTC time zone.  You can also select a start and end date for the flow runs.
+* **Date** - Selector to choose the interval when the export should take place. Best practice is to set your start and end date to line up with the duration of your campaigns in your downstream platforms.  When selecting an export interval, the last day of the interval is not included in the exports. For example, if you select an interval of January 4 - 11, the last file export will take place on January 10.
+<br>  ![Scheduling](../../assets/catalog/advertising/acxiom/image-destination-scheduling.png)
+
+* **File Name Editing** - Allows the user to change the naming of the resulting file on S3, the options presented under this allows for a number of flow derived values, or as a custom text value entered by the user.  Acxiom should be informed of the format.  If you don’t select the Date and Time component, the file names will be static and the new exported file will overwrite the previous file in your storage location with each export.
+  * **Audience name** - The name of the exported audience.
+  * **Destination** - Name of the destination card itself, as it is show in the catalog with spaces replaced with underscores.
+  * **Destination ID** - The ID of the destination dataflow you use to export the audience.
+  * **Destination name** - The name of the destination dataflow you use to export the audience.
+  * **Audience source** - The source identifier for the exported audience
+  * **Audience ID** - The ID of the exported audience
+  * **Organization name** - Your organization name within Experience Platform.
+  * **Sandbox name** - The ID of the sandbox you use to export the audience.
+  * **Date and time** - Select between adding a MMDDYYYY_HHMMSS format or a UNIX 10-digit timestamp of the time when the files are generated. Choose one of these options if you would like your files to have a dynamic file name generated with each incremental export.
+  * **Custom text** - Any custom text that you want to add to the file names.
+
+
+## Mapping ##
```

Comment:
with regard to the mapping step, is there anything particular to this destination that customers would need to know? may be some required or recommended mappings?

Reply:
Added a "Mapping suggestions" section which outlines expectations.